src/ApplicationLogs/LogReader.cs

```diff
@@ -90,14 +90,19 @@ public static JObject TxLogToJson(Blockchain.ApplicationExecuted appExec)
             trigger["vmstate"] = appExec.VMState;
             trigger["exception"] = GetExceptionMessage(appExec.Exception);
             trigger["gasconsumed"] = appExec.GasConsumed.ToString();
-            try
+            var stack = new JArray();
+            foreach (var item in appExec.Stack)
             {
-                trigger["stack"] = appExec.Stack.Select(q => q.ToJson(Settings.Default.MaxStackSize)).ToArray();
-            }
-            catch (Exception ex)
-            {
-                trigger["exception"] = ex.Message;
+                try
+                {
+                    stack.Add(item.ToJson(Settings.Default.MaxStackSize));
+                }
+                catch (Exception ex)
+                {
+                    stack.Add("error: " + ex);
```

Comment:
I dont think will output will help much. Seeing how the only reason it would fail is because its an interop and cant be serialized. Better to put an empty or null stackitem instead.